### PR TITLE
Add documentation on how to modify gcttl by tiup

### PR DESCRIPTION
**Pull request by asddongmen**
Add documentation on how to modify gcttl by tiup.
[#1825](https://github.com/pingcap/ticdc/issues/1825)

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -24,6 +24,35 @@ tiup cluster upgrade <cluster-name> v4.0.6
...
+  pd: {}
+  tiflash: {}
+  tiflash-learner: {}
+  pump: {}
+  drainer: {}
+  cdc:
+    gc-ttl: 3600
+```
+
+修改完毕之后需要执行 tiup cluster relaod 命令来重新加载。
```
Comment (amyangfei) [inline ticdc/manage-ticdc.md:54]:
```suggestion
修改完毕之后需要执行 tiup cluster relaod -R cdc 命令来重新加载。
```

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -24,6 +24,35 @@ tiup cluster upgrade <cluster-name> v4.0.6
...
+
+本部分介绍如何使用 TiUP 的 [tiup cluster edit-config](/tiup/tiup-component-cluster-edit-config.md) 命令来修改 TiCDC 的配置。在以下例子中，假设需要把 TiCDC 的 `gc-ttl` 从默认值 `86400` 修改为1小时。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup cluster edit-config <cluster-name> 
+```
+
+执行以上命令之后，进入到 vi 编辑器页面之后，修改 [server-configs](/tiup/tiup-dm-topology-reference.md#server_configs) 中 cdc 的配置，如下所示：
```
Comment (amyangfei) [inline ticdc/manage-ticdc.md:37]: link is error

Comment (asddongmen) [reply ticdc/manage-ticdc.md]: corrected.

**Review by amyangfei — APPROVED** [review]

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -24,6 +24,35 @@ tiup cluster upgrade <cluster-name> v4.0.6
 * TiCDC v4.0.2 对 `changefeed` 的配置做了调整，请参阅[配置文件兼容注意事项](/ticdc/manage-ticdc.md#配置文件兼容性的注意事项)。
 * 升级期间遇到的问题及其解决办法，请参阅[使用 TiUP 升级 TiDB](/upgrade-tidb-using-tiup.md#4-升级-faq)。
 
+## 使用 TiUP 修改 TiCDC 配置
+
+本部分介绍如何使用 TiUP 的 [tiup cluster edit-config](/tiup/tiup-component-cluster-edit-config.md) 命令来修改 TiCDC 的配置。在以下例子中，假设需要把 TiCDC 的 `gc-ttl` 从默认值 `86400` 修改为1小时。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:29]:
```suggestion
本节介绍如何使用 TiUP 的 [`tiup cluster edit-config`](/tiup/tiup-component-cluster-edit-config.md) 命令来修改 TiCDC 的配置。在以下例子中，假设需要把 TiCDC 的 `gc-ttl` 从默认值 `86400` 修改为 1 小时。
```

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -24,6 +24,35 @@ tiup cluster upgrade <cluster-name> v4.0.6
...
+
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:30]:
```suggestion

首先执行以下命令。将 `<cluster-name>` 替换成实际的集群名。

```

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -24,6 +24,35 @@ tiup cluster upgrade <cluster-name> v4.0.6
...
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup cluster edit-config <cluster-name> 
+```
+
+执行以上命令之后，进入到 vi 编辑器页面之后，修改 [server-configs](/tiup/tiup-cluster-topology-reference.md#server_configs) 中 cdc 的配置，如下所示：
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:37]:
```suggestion
执行以上命令之后，进入到 vi 编辑器页面，修改 [`server-configs`](/tiup/tiup-cluster-topology-reference.md#server_configs) 下的 `cdc` 配置，如下所示：
```

File: ticdc/manage-ticdc.md [hunk 6]
```diff
@@ -24,6 +24,35 @@ tiup cluster upgrade <cluster-name> v4.0.6
...
+
+{{< copyable "shell-regular" >}}
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:39]:
```suggestion
```

File: ticdc/manage-ticdc.md [hunk 7]
```diff
@@ -24,6 +24,35 @@ tiup cluster upgrade <cluster-name> v4.0.6
...
+  pd: {}
+  tiflash: {}
+  tiflash-learner: {}
+  pump: {}
+  drainer: {}
+  cdc:
+    gc-ttl: 3600
+```
+
+修改完毕之后需要执行 tiup cluster relaod -R cdc 命令来重新加载。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:54]:
```suggestion
修改完毕后执行 `tiup cluster relaod -R cdc` 命令重新加载配置。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**